pyccel/codegen/printing/ccode.py

```diff
@@ -1,6 +1,11 @@
 # coding: utf-8
 # pylint: disable=R0201
 
+from sympy.core import Tuple
+from pyccel.ast.numbers   import BooleanTrue
+from pyccel.ast.builtins import PythonTuple
```

Comment:
See above

Reply:
`Tuple` and `PythonTuple` were used before I removed the condition above, they are no longer being used